.github/test-spec.yml

```diff
@@ -517,3 +517,27 @@
       - "subsys/nrf_security/**/*"
       - "!subsys/nrf_security/doc/**/*"
       - "!subsys/nrf_security/*.rst"
+
+"CI-suit-dfu-test":
```

Comment:
Missing entries:
  - "subsys/mgmt/suitfu/**/*"
  - "subsys/nrf_rpc/**/*"
  - "samples/suit/**/*"
  - "tests/subsys/suit/**/*"
  - "lib/dk_buttons_and_leds/**/*"
  - "subsys/nrf_security/**/*"
I'm not sure if all added entries are valid and should trigger SUIT tests. For example trusted-firmware-m, is there something the modules/trusted-firmware-m that SUIT depends on? The sampe situation for subsys/bootloader and subsys/nonsecure. Can we have post-daily online review with the dev team to confirm what is required from their perspective?

Reply:
Disscussion regaring trusted-firmware-m: https://github.com/nrfconnect/sdk-nrf-next/pull/2085#pullrequestreview-1786144402

Reply:
Thanks, updated on post-daily meeting.